devel/ncurses/files/hex.diff

```diff
@@ -1,20 +0,0 @@
---- misc/terminfo.src.orig	2015-07-26 05:27:20.000000000 +1000
-+++ misc/terminfo.src	2015-08-17 15:05:08.000000000 +1000
-@@ -7408,7 +7408,7 @@
- # hp2621 with function labels. Most of the time they are off,
- # but inside vi, the function key labels appear. You have to
- # hold down shift to get them to xmit.
--hp2621|hp2621a|hp2621A|2621|2621a|2621A|hp2621-wl|2621-wl|hp 2621 w/labels,
-+hp2621|hp2621a|2621|2621a|hp2621-wl|2621-wl|hp 2621 w/labels,
```

Comment:
You're right, my oversight: this should have stayed. I missed that it was protecting against case-insensitivity in the filesystem. I restored and refreshed the patch.